app/design/frontend/Magento/blank/web/css/source/_layout.less

```diff
@@ -59,6 +59,10 @@
         padding-right: @layout__width-xs-indent;
     }
 
+    .page-header {
+        .lib-sticky-nav();
+    }
```

Comment:
Just as a note, _most_ themes inherit from blank.

Adding this much new functionality to the root theme scares me, personally. Seems liable to immediately cause issues in the likely hundreds of child themes. There should definitely be a conversation around this before merge IMO